lambda-cleanup.tf

```diff
@@ -0,0 +1,36 @@
+# Trigger:
+#	Cloudwatch cron schedule
+# Resources:
+#	KMS
+#	RDS
+#	Secret manager secrets
+#	SSM parameters
+
+module "cleanup" {
+  source = "./modules/lambda"
+  enable = true
+  name   = format("%s-cleanup", module.labels.id)
+
+  aws_parameter_arns = [
+    aws_ssm_parameter.db_database.arn,
+    aws_ssm_parameter.db_host.arn,
+    aws_ssm_parameter.db_port.arn,
+    aws_ssm_parameter.db_reader_host.arn,
+    aws_ssm_parameter.db_ssl.arn,
+    aws_ssm_parameter.security_code_lifetime_mins.arn,
+    aws_ssm_parameter.upload_token_lifetime_mins.arn
+  ]
+  aws_secret_arns                = [data.aws_secretsmanager_secret_version.rds_read_write.arn]
+  cloudwatch_schedule_expression = var.cleanup_schedule
+  config_var_prefix              = local.config_var_prefix
+  handler                        = "cleanup.handler"
+  kms_writer_arns                = [aws_kms_key.sqs.arn]
```

Comment:
Should this be kms_reader_arns ? or do we even need - do we do something with SQS in this lamda ?

Reply:
No, these are not required - didn't realise they were for SQS

Reply:
Will remove